content/site.json

```diff
@@ -110,10 +110,18 @@
           "isLive": true
         },
         {
-          "name": "UdL Research Data Space",
-          "link": "https://dataspace.angliru.udl.cat/",
-          "description": "An open research platform following the FAIR guiding principles of findability, accessibility, interoperability and reusability, while guaranteeing data sovereignty.",
-          "image": "udl-logo.webp",
+          "name": "AgrospAI",
+          "link": "https://agrospai.udl.cat/",
+          "description": "Agri-food Data Space Demonstrator for Sovereign Data-Sharing and Artificial Intelligence Services.",
+          "image": "agrospai-logo.webp",
+          "category": "agriculture",
+          "isLive": true
+        },
+        {
+          "name": "Agrifood TEF",
+          "link": "http://dataspace.agrifoodtef.eu/",
```

Comment:
issue(blocking): Please update and use https here.

Reply:
Done, removed till they enable HTTPS